tests/sca_image/test_runner.py

```diff
@@ -65,3 +66,45 @@ def test_runner_honors_enforcement_rules(mock_bc_integration, image_name, cached
     assert summary["failed"] == 0
     assert summary["skipped"] == 3
     assert summary["parsing_errors"] == 0
+
+
+@responses.activate
+def test_licenses_status(mock_bc_integration):
```

Comment:
Great test! WDYT about creating a test for a case the request fails?

Reply:
sure!